lib/dest/write-contents/index.js

```diff
@@ -38,26 +40,13 @@ function writeContents(opt) {
     // This is invoked by the various writeXxx modules when they've finished
     // writing the contents.
     function onWritten(writeErr) {
-      if (isErrorFatal(writeErr)) {
+      if (fo.isOverwriteError(writeErr, file.flag)) {
```

Comment:
I think this is the inverse of naming/usage.  I believe it's **not** an overwrite error if it returns `true`.  Maybe I'm wrong?